### ISSUE TITLE
Can/should the polling mechanism for projectors take into account snapshots and not query the whole event history?

### ISSUE THREAD

**Issue by TomKaltz** [Rotorsoft/eventually-monorepo#65]
I may be reading the code wrong but the polling mechanism seems to do a query of the whole event history without trying to load from snapshot first. Is this intended?

**Comment by Rotorsoft** [comment]
can you point to the source code?

**Comment by TomKaltz** [comment]
Actually looking deeper it makes sense the subscription polling would load from the watermark and not the snapshot although now I'm kind of curious if that could be implemented as a possible performance enhancement.  Seems like not, since the projector should probably see every event in sequence regardless of aggregate snapshotting.

The reason I started looking into this is because I was showing an immense amount of reads on my testing firestore database.  Then I realized that I'm also not using snapshotting for one of my test aggregates since I was running into concurrency errors.  That would explain the high amount of records read.  It tips me off that my aggregate boundary is probably incorrect.
